app/graphql/mutations/base_mutation.rb

```diff
@@ -1,4 +1,8 @@
 module Mutations
-  class BaseMutation < GraphQL::Schema::RelayClassicMutation
+  class BaseMutation < GraphQL::Schema::Mutation
```

Comment:
what's the **TL;DR** difference between `RelayClassMutation` and `Mutation`?

Reply:
`RelayClassMutation` expects a certain form of return type, but we don't follow that form because we don't follow relay standards currently.